time_stamper_cv/include/configuration.h

```diff
@@ -41,10 +41,13 @@ struct LedRowConfig {
 typedef std::map<std::string, LedRowConfig> LedRowConfigRepository;
 
 struct ImageProcessorConfig {
+  inline static constexpr const char *TOP_ROW{"TopRow"};
```

Comment:
I guess you used constexpr's instead of enums so that you can get the string name value of it as well?